src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const repaired = {};
```

Comment:
What was repaired ? It's better to use explicit namings
```suggestion
  const repairedRobot = {};
```